controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2078,3 +2067,25 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
 
     return 0;
 }
+
+const std::string cli_bml::string_from_int_array(uint8_t *arr, size_t arr_max_size)
+{
+    std::stringstream ss;
+    if (arr) {
+        for (size_t i = 0; i < arr_max_size; i++) {
+            ss << ((i != 0) ? ", " : "") << static_cast<int>(arr[i]);
```

Comment:
In this case, we want `int(arr[i])` instead of static_cast, I think. Or better yet, `unsigned int(arr[i])`.

Reply:
Changed